e2e/support/commands/ui/icon.js

```diff
@@ -1,3 +1,15 @@
-Cypress.Commands.add("icon", icon_name => {
-  cy.get(`.Icon-${icon_name}`);
-});
+Cypress.Commands.add(
```

Comment:
Previously, we don't take the previous subject into account. That makes statements like this never work. (Please see the attach screenshot of the situation below)
```
cy.findByText("Text").icon("gear").click();
```
This change will make it more convenient so that we don't need to always wrap `cy.icon()` within `.within(() => {})` every time.

<img width="232" alt="image" src="https://user-images.githubusercontent.com/1937582/233363721-9e25391a-887f-4619-a409-a48910475331.png">